<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Valid Login is Failed]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="29458" started-at="2017-11-06T05:53:33Z" finished-at="2017-11-06T05:54:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29458" started-at="2017-11-06T05:53:33Z" finished-at="2017-11-06T05:54:02Z">
      <class name="com.script.actiTime.Script1">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.script.actiTime.Script1@224edc67]" name="openBrowser" is-config="true" duration-ms="8126" started-at="2017-11-06T11:23:33Z" finished-at="2017-11-06T11:23:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="validLogin()[pri:0, instance:com.script.actiTime.Script1@224edc67]" name="validLogin" duration-ms="20577" started-at="2017-11-06T11:23:41Z" finished-at="2017-11-06T11:24:01Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.fail(Assert.java:100)
	at com.script.actiTime.Script1.validLogin(Script1.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Valid Login is Failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- validLogin -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.script.actiTime.Script1@224edc67]" name="closeBrowser" is-config="true" duration-ms="708" started-at="2017-11-06T11:24:01Z" finished-at="2017-11-06T11:24:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.script.actiTime.Script1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
